cypress/page/shopping-cart.page.ts

```diff
@@ -2,7 +2,7 @@ class ShoppingCartPage {
   private proceedToCheckoutBtn: string;
 
   constructor() {
-    this.proceedToCheckoutBtn = ".cart_navigation span";
+    this.proceedToCheckoutBtn = ".cart_navigation .button";
```

Comment:
see if you can use an attribute to indicate that we are looking for the "Proceed to checkout" button